src/server_name.rs

```diff
@@ -74,6 +74,21 @@ impl<'a> fmt::Debug for ServerName<'a> {
     }
 }
 
+#[cfg(feature = "alloc")]
+impl TryFrom<String> for ServerName<'static> {
```

Comment:
I think we should update the `ServerName` doc block to reference this alongside the existing `ServerName::try_from(&str)`

Reply:
Added. Please take a look; feel free to merge and publish if satisfied.

Reply:
Thanks, will do